python/private/py_executable.bzl

```diff
@@ -834,14 +835,14 @@ def _create_zip_file(ctx, *, output, zip_main, runfiles):
     manifest.add("__main__.py={}".format(zip_main.path))
     manifest.add("__init__.py=")
     manifest.add(
-        "{}=".format(
-            _get_zip_runfiles_path("__init__.py", workspace_name, legacy_external_runfiles),
-        ),
+        "{}=".format(_get_zip_runfiles_path(_INIT_PY, workspace_name)),
     )
 
     def map_zip_empty_filenames(list_paths_cb):
         return [
-            _get_zip_runfiles_path(path, workspace_name, legacy_external_runfiles) + "="
+            # FIXME @aignas 2025-12-06: what kind of paths do we expect here? Will they
```

Comment:
I would expect the paths to match what you'd see for a file in `runfiles.files`.

e.g., if a file in the runfiles comes from an external repo (and would have a short path of `external/repo/foo.py` or `../repo/foo.py`), then there would be an empty file with a similar short path.